openfe/protocols/openmm_md/plain_md_methods.py

```diff
@@ -0,0 +1,604 @@
+        )
+        # equilibrate
+        # NVT equilibration
+
+        if verbose:
+            logger.info("Running NVT equilibration")
+
+        # Set barostat frequency to zero for NVT
+        for x in simulation.context.getSystem().getForces():
+            if x.getName() == 'MonteCarloBarostat':
+                x.setFrequency(0)
+
+        simulation.context.setVelocitiesToTemperature(
+            to_openmm(thermo_settings.temperature))
+
+        t0 = time.time()
+        simulation.step(equil_steps_nvt)
+        t1 = time.time()
+        if verbose:
+            logger.info(
+                f"Completed NVT equilibration in {t1 - t0} seconds")
+
+        # Save last frame NVT equilibration
+        positions = to_openmm(
+            from_openmm(simulation.context.getState(
+                getPositions=True, enforcePeriodicBox=False
+            ).getPositions()))
+
+        traj = mdtraj.Trajectory(
+            positions[selection_indices, :],
+            mdtraj_top.subset(selection_indices),
+        )
+        traj.save_pdb(
+            shared_basepath / simulation_settings.equil_NVT_structure
+        )
+
+        # NPT equilibration
+        if verbose:
+            logger.info("Running NPT equilibration")
+        simulation.context.setVelocitiesToTemperature(
+            to_openmm(thermo_settings.temperature))
+
+        # Enable the barostat for NPT
+        for x in simulation.context.getSystem().getForces():
+            if x.getName() == 'MonteCarloBarostat':
+                x.setFrequency(integrator_settings.barostat_frequency.m)
+
+        t0 = time.time()
+        simulation.step(equil_steps_npt)
+        t1 = time.time()
+        if verbose:
+            logger.info(
+                f"Completed NPT equilibration in {t1 - t0} seconds")
+
+        # Save last frame NPT equilibration
+        positions = to_openmm(
+            from_openmm(simulation.context.getState(
+                getPositions=True, enforcePeriodicBox=False
+            ).getPositions()))
+
+        traj = mdtraj.Trajectory(
+            positions[selection_indices, :],
+            mdtraj_top.subset(selection_indices),
+        )
+        traj.save_pdb(
+            shared_basepath / simulation_settings.equil_NPT_structure
+        )
+
+        # production
+        if verbose:
+            logger.info("running production phase")
+
+        # Setup the reporters
+        simulation.reporters.append(XTCReporter(
+            file=str(shared_basepath / simulation_settings.output_filename),
+            reportInterval=simulation_settings.trajectory_interval.m,
+            atomSubset=selection_indices))
+        simulation.reporters.append(openmm.app.CheckpointReporter(
+            file=str(shared_basepath / simulation_settings.checkpoint_storage),
+            reportInterval=simulation_settings.checkpoint_interval.m))
+        simulation.reporters.append(openmm.app.StateDataReporter(
```

Comment:
TODO: as discussed in a call a few weeks back, in a next PR we should hook up the results object to point to these and maybe have methods for reading data out for commonly needed things (reduced potential etc..)